src/Containers.jsx

```diff
@@ -223,8 +230,16 @@ const ContainerActions = ({ container, onAddNotification, version, localImages }
             <DropdownItem key="start"
                           onClick={() => startContainer()}>
                 {_("Start")}
-            </DropdownItem>
+            </DropdownItem>,
```

Comment:
Why the trailing comma here? previous stanzas don't use it either, and it looks like an error. This isn't a list.